web/templates/base.html

```diff
@@ -89,5 +89,6 @@
 		<script type="text/javascript" src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 		<script type="text/javascript" src="{% static 'js/countdown.js' %}"></script>
 		{% block custom_js %}{% endblock custom_js %}
+		<script type="text/javascript" src="{% static 'js/script.js' %}"></script>
```

Comment:
custom_js should come after script.js (so you can override it from child template)